scopes/design/ui/external-link/external-link.docs.mdx

```diff
@@ -0,0 +1,21 @@
+---
+labels: ['react', 'typescript', 'link', 'external']
+description: 'A link component to display an external link with no text decoration'
+---
+
+import { ThemeCompositions } from '@teambit/documenter.theme.theme-compositions';
+import { ExternalLink } from './external-link';
+
+### Overview
+
+A link component for attaching a link to text without text decoration
+
+```js live
+() => {
+  return (
+    <ThemeCompositions style={{ minHeight: 120}}>
```

Comment:
No need to wrap the example with ThemeComposition, because the font is globally in this page.
We need to use it only in the compositions file.